composer.json

```diff
@@ -32,7 +32,7 @@
         "doctrine/orm": "^2.13 || ^3.0",
         "doctrine/phpcr-odm": "^1.8 || ^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.12 || ^2.0",
-        "phpunit/phpunit": "^9.6",
+        "phpunit/phpunit": "^10.5.29",
```

Comment:
Please remove the patch version from the requirement

Reply:
Fixed